magento/adobe-stock-integration
# Fix  Open Images uploaded outside of media gallery are not reflected …

…in the media gallery





### Description (*)


### Fixed Issues (if relevant)

1. Fixes magento/adobe-stock-integration#1428: Images uploaded outside of media gallery are not reflected in the media gallery
2. Fixes magento/adobe-stock-integration#1435: [MFTF] Cover uploading images outside of the Media Gallery


### Manual testing scenarios (*)
1. Login to Admin panel
2. Go to Catalog -> Category
3. Upload the Category image using the "Upload button"
4. Save the category
5. Open Media Gallery (i.e. using "Select from Gallery" button for the same Category image field)
6. Go to catalog/category folder in the Media Gallery

**The image saved for the category should be present in the media gallery**
